build.gradle

```diff
@@ -15,11 +15,12 @@ group = "com.splunk.public"
 boolean snapshot = getProperty("snapshot") == 'yes'
 
 def versions = [
-    'opentelemetry'         : snapshot ? "0.17.0-SNAPSHOT"       : "0.16.0",
-    'opentelemetryAlpha'    : snapshot ? "0.17.0-alpha-SNAPSHOT" : "0.16.0-alpha",
-    'opentelemetryJavaagent': snapshot ? "0.17.0-SNAPSHOT"       : '0.16.1',
-    'mockito'               : '3.8.0',
-    'jupiter'               : '5.7.1'
+    'opentelemetry'              : snapshot ? "0.18.0-SNAPSHOT"       : "1.0.0",
+    'opentelemetryAlpha'         : snapshot ? "0.18.0-alpha-SNAPSHOT" : "1.0.0-alpha",
+    'opentelemetryJavaagent'     : snapshot ? "0.18.0-SNAPSHOT"       : '1.0.0',
+    'opentelemetryJavaagentAlpha': snapshot ? "0.18.0-alpha-SNAPSHOT" : '1.0.0-alpha',
```

Comment:
Certainly we already have 1.0.0-SNAPSHOT?

Reply:
Fixed!